MobiFlight/MobiFlightModule.cs

```diff
@@ -725,7 +722,7 @@ void OnDebugPrint(ReceivedCommand arguments)
             String value1 = arguments.ReadStringArg();
             String value2 = arguments.ReadStringArg();
             String value3 = arguments.ReadStringArg();
-            Log.Instance.log($"{this.Name}.debug: Firmware -> {value1} {value2} {value3}.", LogSeverity.Debug);
+            Log.Instance.log($"{this.Name}.debug: Firmware -> {value1} {value2} {value3}", LogSeverity.Debug);

```

Comment:
Removed the period since it looked really weird in logs when only one value was sent from the firmware.